### DISCUSSION TITLE
Idea: a page with a PSA on withdrawing nano from exchanges

### DISCUSSION THREAD

**Discussion by joewelow** [mistakia/nano-community#42] [Proposals]
Given that community education is key to keeping Nano decentralized, I propose we create a page dedicated to explaining why holders are strongly encouraged to withdraw from Binance/other exchanges and delegate to a good representative.

It could be another link in the **Guides** menu, perhaps called `Why Withdraw?`. Also, another idea was to have a URL redirect to the page: `nano.community/withdraw` redirecting to `nano.community/getting-started-users/withdraw` (for consistency). That way, it is easier to link in a reddit comment/Discord/etc.

Please let me know your thoughts!

**Comment by mistakia** [comment]
Agree with both ideas. Not sure about the terminology but I don't really have any better ideas 😅

Currently, there is only this section: https://nano.community/getting-started-users/basics#securing-the-network

That should be expanded to have its own dedicated page.

Content to consider including:
- https://www.reddit.com/r/nanocurrency/comments/o3brpp/newcomers_move_your_nanos_off_exchange_and_change/
- https://medium.com/nanocurrency/why-you-should-take-your-nano-off-exchanges-for-yourself-the-network-and-nanos-value-b1890b01136c

**Comment by mistakia** [comment]
Merged https://github.com/mistakia/nano-community/pull/43
- Added page "choosing a representative" (https://nano.community/getting-started-users/choosing-a-representative)
- Elevated self-custody vs custodial implications on storing page (https://nano.community/getting-started-users/storing/basics#self-custody-vs-custodial)
